data/generatingScripts/evaluateResults.m

```diff
@@ -0,0 +1,27 @@
+function [statistic, vectors] = evaluateResults(anomaly, has_anomaly, tresh_hold)
+% evaluateResults: [vectors statistics] = evaluateResults(anomaly, has_anomaly)
+% anomaly ... float: obtained anomaly, computed by an algortihm [Nx1]
+% has_anomaly ... bool: anotation of original data [Nx1]
+% tresh_hold ... float: tresh hold to determind if anomaly(i) should be
+%                 considered as anomaly or not.
+% OUTPUTS:
+% vectors.FP/.FN/.TP/.TN vectors of bools showing where those occured
+% statistic.FP/.FN/.TP/.TN float,a percentage of the whole set
+% statistic.tresh_hold 
+
+assumed_anomaly = anomaly >= tresh_hold;
+
+vectors.TP = (assumed_anomaly == 1) & (has_anomaly == 1);
+vectors.TN = (assumed_anomaly == 0) & (has_anomaly == 0);
+vectors.FP = (assumed_anomaly == 1) & (has_anomaly == 0);
+vectors.FN = (assumed_anomaly == 0) & (has_anomaly == 1);
```

Comment:
This is a nice functionality! :+1: 
Maybe we can compute precision/recall/F right away here too?